elide-async/src/main/java/com/yahoo/elide/async/service/DefaultResultStorageEngine.java

```diff
@@ -68,93 +69,117 @@ public URL storeResults(String asyncQueryID, String responseBody) {
             throw new IllegalStateException(e);
         }
 
+        AsyncQueryResultStorage aQRS = null;
+        try {
+            aQRS = (AsyncQueryResultStorage) DBUtil.executeInTransaction(elide, dataStore,
+                    (tx, scope) -> {
+
+                        EntityProjection asyncQueryCollection = EntityProjection.builder()
+                                .type(AsyncQueryResultStorage.class)
+                                .build();
+
+                        Object loaded = tx.loadObject(asyncQueryCollection, asyncQueryResultStorage.getId(), scope);
```

Comment:
This is going to return null cos the AsyncQueryResultStorage object does not exist in the database. We should be creating a new AsyncQueryResultStorage with asyncQueryID and response. Then we should just call tx.save for that object.